modules/javafx.base/src/test/java/test/javafx/collections/VetoableObservableListTest.java

```diff
@@ -205,12 +205,24 @@ public void testAddAll() {
         assertSingleCall(new String[] {"a", "b"}, new int[] {4,4});
     }
 
+    @Test
+    public void testAddAll_subList() {
+        list.addAll(list.subList(0, 2));
+        assertSingleCall(new String[] {"foo", "bar"}, new int[] {4, 4});
+    }
```

Comment:
suggestion: also check that the list contains the newly added elements?
(here and in added tests that involve subList?)

Reply:
I've added checks for the list content in all modified tests.